src/main/java/cart/domain/coupon/discountpolicy/FixedDiscountPolicy.java

```diff
@@ -0,0 +1,21 @@
+package cart.domain.coupon.discountpolicy;
+
+import cart.domain.Money;
+import java.math.BigDecimal;
+
+public class FixedDiscountPolicy implements DiscountPolicy {
+
+    @Override
+    public boolean isValid(BigDecimal discountValue) {
+        boolean isGreaterThanZero = discountValue.compareTo(BigDecimal.ZERO) == 1;
+        return isGreaterThanZero;
```

Comment:
```suggestion
        return discountValue.compareTo(BigDecimal.ZERO) > 0;
```
위 처럼 간결하게 바꿔보면 어떨까요? 전체적으로 비슷한 메소드들이 많은데 다같이 바꿔보면 좋을것 같아요!

Reply:
이 부분은 compareTo 메서드가 코드를 한번에 이해하는데 어렵다고 느껴져서 변수로 빼보았습니다!

Reply:
compareTo와 같이 오래된 인터페이스에 대해서는 굳이 변수로 도출하지 않고 간결하게 작성하는게 더 좋아보이는데요.
해당 정책을 사용하는 클라이언트 입장에서는 메소드 이름으로 충분히 어떤 동작을 할것임을 기대했기 때문에 내부 로직을 더 간결하게 작성해보는게 좋아보여요.